yarn-project/end-to-end/src/bench/client_flows/transfers.test.ts

```diff
@@ -146,10 +146,10 @@ describe('Transfer benchmark', () => {
              * We should have created the following nullifiers:
              * - One per created note
              * - One for the transaction
-             * - One for the fee note if we're using private fpc
+             * - One for the fee note and one for the partial note validity commitment if we're using private fpc
              */
             expect(txEffects!.data.nullifiers.length).toBe(
-              notesToCreate + 1 + (benchmarkingPaymentMethod === 'private_fpc' ? 1 : 0),
+              notesToCreate + 1 + (benchmarkingPaymentMethod === 'private_fpc' ? 2 : 0),
```

Comment:
Interesting that we do not fully capture DA usage here (e.g. we don't see how this reduces pub storage writes). @Thunkar perhaps we could find this information in a local simulation?